### PR TITLE
Adds ChainerRL Bibtex to the README

### PR DESCRIPTION
**Pull request by prabhatnagarajan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,3 +114,18 @@ Any kind of contribution to ChainerRL would be highly appreciated! If you are in
...
+
+## Citations
+
+If you use ChainerRL for your research, we ask that you cite the following paper:
+
+```
+@InProceedings{fujita2019chainerrl,
+  author = {Fujita, Yasuhiro and Kataoka, Toshiki and Nagarajan, Prabhat and Ishikawa, Takahiro},
+  title = {ChainerRL: A Deep Reinforcement Learning Library},
+  booktitle = {NeurIPS 2019 Deep Reinforcement Learning Workshop},
```
Comment (toslunar) [inline README.md]: It's possible that `booktitle` starts with "Proceedings of" and contains "33rd", etc.  Could you check the formal title of the proceedings?

Comment (prabhatnagarajan) [reply README.md]: There is no formal title for workshop proceedings.

Comment (prabhatnagarajan) [reply README.md]: Or are you referring to the formal NeurIPS proceedings?

Comment (toslunar) [reply README.md]: Thanks.  I got it.

File: README.md [hunk 2]
```diff
@@ -114,3 +114,18 @@ Any kind of contribution to ChainerRL would be highly appreciated! If you are in
 ## License
 
 [MIT License](LICENSE).
+
+## Citations
+
+If you use ChainerRL for your research, we ask that you cite the following paper:
```
Comment (muupan) [inline README.md]: I think it's not good to *ask* for citation.

Instead we can just write `To cite ChainerRL in publications:` as https://github.com/openai/baselines

Comment (prabhatnagarajan) [reply README.md]: OK. Done.

**Pull request merged**